app-creator/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"log"
+	"os"
+
```

Comment:
Intentional extra space?

Reply:
Handled by the editor, separates stdlib imports from external ones.